### PR TITLE
Editorial: prefer shorthand RIA

### PR DESCRIPTION
**Pull request by UltCombo**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -26234,8 +26231,7 @@
           1. If no arguments were passed to this function invocation, let _s_ be `""`.
           1. Else,
             1. If NewTarget is *undefined* and Type(_value_) is Symbol, return SymbolDescriptiveString(_value_).
-            1. Let _s_ be ToString(_value_).
-          1. ReturnIfAbrupt(_s_).
+            1. Let _s_ be ? ToString(_value_).
```
Comment (UltCombo) [inline spec.html]: I believe this can never throw, so perhaps it is worth removing the RIA completely.

Comment (UltCombo) [inline spec.html]: Oh never mind the comment above, `new String(Symbol())` will throw. So the RIA should stay.

**Comment by bterlson** [comment]
Thanks for finding these! It looks like many of these were missed as a result of improper ecmaspeak call syntax - there should not be a space between the abstract op id and the following parenthesis. @UltCombo can you fix those as well in this commit?

**Comment by UltCombo** [comment]
Sure, done.
I've also fixed a few more call syntax ecmaspeak violations that I found.

**Comment by UltCombo** [comment]
Rebased on master.

**Comment by bterlson** [comment]
Committed as 5921ef9. Thanks so much!

**Comment by UltCombo** [comment]
No problem. :smile:

I believe there are more of those space-between-abstract-operation-and-parenthesis violations. I wonder if we can automate finding those.

**Pull request closed without merging**